spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-gcp-starters</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-gcp-starter-core</artifactId>
+	<name>Spring Cloud GCP Core Starter</name>
+	<description>Starter for Google Cloud</description>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
+</project>
```

Comment:
You should consider to let your IDEA to take care about these:

![line_feed_in_the_end](https://cloud.githubusercontent.com/assets/1071590/26468753/8dbb282c-4165-11e7-98fe-3f0682aaf3f5.png)